easybuild/easyconfigs/p/pkgconfig/pkgconfig-1.5.5-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'pkgconfig'
+version = '1.5.5'
```

Comment:
@SebastianAchilles As discussed in Slack, the `-python` versionsuffix was there to avoid confusion between `pkg-config` (the tool) and `pkgconfig` (the Python wrappers), so perhaps that be reinstated?

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/14426